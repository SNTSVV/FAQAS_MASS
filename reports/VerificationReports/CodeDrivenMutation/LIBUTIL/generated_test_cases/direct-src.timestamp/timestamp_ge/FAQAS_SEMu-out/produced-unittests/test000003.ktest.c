
/* Wrapping main template for the function timestamp_ge defined in the file /opt/faqas_semu_fork/case_studies/LIBUTIL/util_codes/../WORKSPACE/DOWNLOADED/libutil/src/timestamp.c */
/* The following mutants (IDs) were targeted to generated this test: [6] */

#include <stdio.h>
#include <string.h>



int main(int argc, char** argv)
{
    (void)argc;
    (void)argv;

    // Declare variable to hold function returned value
    int result_faqas_semu;

    // Declare arguments and make input ones symbolic
    gs_timestamp_t base;
    gs_timestamp_t test;
    memset(&base, 0, sizeof(base));
    memset(&test, 0, sizeof(test));
    const unsigned char base_faqas_semu_test_data[] = {0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00};
    const unsigned char test_faqas_semu_test_data[] = {0x00, 0x00, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00};
    memcpy(&base, base_faqas_semu_test_data, sizeof(base));
    memcpy(&test, test_faqas_semu_test_data, sizeof(test));

    // Call function under test
    result_faqas_semu = timestamp_ge(&base, &test);

    // Make some output
    printf("FAQAS-SEMU-TEST_OUTPUT: result_faqas_semu = %d\n", result_faqas_semu);
    return (int)result_faqas_semu;
}