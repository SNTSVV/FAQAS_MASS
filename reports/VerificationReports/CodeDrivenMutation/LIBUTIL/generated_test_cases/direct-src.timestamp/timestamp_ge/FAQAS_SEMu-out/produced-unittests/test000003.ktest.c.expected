FAQAS-SEMU-TEST_OUTPUT: result_faqas_semu = 1
